app/services/limesurvey_service.rb

```diff
@@ -4,14 +4,18 @@ class LimesurveyService
   BASEURL = 'https://limesurvey.bula21.ch/index.php'
   ADMIN_REMOTECONTROL_URL = URI.parse(BASEURL + '/admin/remotecontrol')
 
-  attr_reader :session_key
   def initialize(username = ENV['LIMESURVEY_USERNAME'],
                  password = ENV['LIMESURVEY_PASSWORD'],
                  survey_id = ENV['LIMESURVEY_SURVEY_ID'])
-    @session_key = fetch_session_key(username, password)
```

Comment:
fetch_session_key always makes a request. We don't need the session in every case, so it should be created lazily